includes/functions.php

```diff
@@ -1854,6 +1854,11 @@ function ap_sanitize_unslash( $str, $from = false, $default = '' ) {
 		return array_map( 'sanitize_text_field', $str );
 	}
 
+	// preserving line breaks for the email notification message
```

Comment:
Hi @rahularyan,
I do think it would be the best scenario, but I don't see how it can be done. Any suggestions?